examples/kubernetes/static_provisioning/README.md

```diff
@@ -6,7 +6,7 @@ This example shows how to make a static provisioned Mountpoint for S3 persistent
 - `non_root.yaml` - same as above, but the pod is spawned as non-root (uid `1000`, gid `2000`)
 - `s3_express_specify_az.yaml` - same as above, but this uses a [S3 Express One Zone](https://docs.aws.amazon.com/AmazonS3/latest/userguide/s3-express-one-zone.html) directory bucket and shows how to specify the availability zone (AZ) of the [persistent volume](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#node-affinity) to co-locate the pods with the bucket for lower latency access
 - `multiple_buckets_one_pod.yaml` - same as above, with multiple buckets mounted in one pod. Note: when mounting multiple buckets in the same pod, the `volumeHandle` must be unique as specified in the [CSI documentation](https://kubernetes.io/docs/concepts/storage/volumes/#csi).
-- `multiple_pods_one_pv.yaml` - same as above, with multiple pods mounting the same persistent volume, in Deployment kind, that can be used to scale pods, using HPA or other method to create more replicas. This example also demonstrates the Pod Sharing feature, where multiple workloads can share a single Mountpoint instance for better resource utilization. See [Pod Sharing](../../docs/FEATURES.md#pod-sharing) for more details.
+- `multiple_pods_one_pv.yaml` - same as above, with multiple pods mounting the same persistent volume, in Deployment kind, that can be used to scale pods, using HPA or other method to create more replicas. This example also demonstrates the Mountpoint Pod Sharing feature, where multiple workloads can share a single Mountpoint instance for better resource utilization. See [Mountpoint Pod Sharing](../../docs/MOUNTPOINT_POD_SHARING.md) for more details.
 - `caching.yaml` - shows how to configure mountpoint to use a cache directory. See the [Mountpoint documentation](https://github.com/awslabs/mountpoint-s3/blob/main/doc/CONFIGURATION.md#caching-configuration) for more details on caching options. Please thumbs up [#11](https://github.com/awslabs/mountpoint-s3-csi-driver/issues/141) or add deatils about your use case if you want improvements in this area.
```

Comment:
Noticed that `caching.yaml` still has `cache /tmp/s3-pv1-cache` which is the old way of configuring cache.